### PR TITLE
USHIFT-1524: migrate pvc-resize test to RF

### PR DESCRIPTION
**Pull request by eslutsky**

### CODE REVIEW

File: test/resources/oc.resource [hunk 1]
```diff
@@ -66,3 +66,10 @@ Oc Expose
     [Arguments]    ${opts}
     ${output}=    Run With Kubeconfig    oc expose ${opts}
     RETURN    ${output}
+
+Oc Patch
+    [Documentation]    Run 'oc patch' on a specific resource in the curret test namespace
+    ...    Returns the command's combined STDOUT/STDER
```
Comment (ggiguash) [inline test/resources/oc.resource:72]: I also see the `curret` misspelled above.
```suggestion
    [Documentation]    Run 'oc patch' on a specific resource in the current test namespace
    ...    Returns the combined STDOUT/STDER output of the command.
```

File: test/suites/storage/pvc-resize.robot [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Resource            ../../resources/common.resource
+Resource            ../../resources/oc.resource
+Resource            ../../resources/ostree-health.resource
+
+Suite Setup         Setup Suite With Namespace
+
+
+*** Variables ***
+${SOURCE_POD}           ./assets/reboot/pod-with-pvc.yaml
```
Comment (ggiguash) [inline test/suites/storage/pvc-resize.robot:12]: Do we need to delete the pod-with-pvc.yaml file from e2e/assets directory?

File: test/suites/storage/pvc-resize.robot [hunk 3]
```diff
@@ -0,0 +1,35 @@
+*** Settings ***
+Documentation       Tests related to a functional PVC after reboot
+
+Resource            ../../resources/common.resource
+Resource            ../../resources/oc.resource
+Resource            ../../resources/ostree-health.resource
+
+Suite Setup         Setup Suite With Namespace
```
Comment (pmtk) [inline test/suites/storage/pvc-resize.robot:8]: Should we have a corresponding `Suite Teardown`? i.e. `Teardown Suite With Namespace`

**Pull request merged**